dae/dae/genomic_resources/genomic_scores.py

```diff
@@ -435,6 +435,10 @@ def get_schema():
                     "index": {"type": "integer"},
                     "name": {"type": "string", "excludes": "index"}
                 }},
+                "chrom_mapping": {"type": "dict", "schema": {
+                    "add_prefix": {"type": "string"},
+                    "del_prefix": {"type": "string", "excludes": "add_prefix"}
+                }}
```

Comment:
`chrom_mapping` allows passing a tab-separated file that enumerates the mapping of individual chromosomes. Check https://github.com/iossifovlab/gpf/blob/e708a722edec28e2d54bed7f718b24c0219ba3b9/dae/dae/genomic_resources/genome_position_table.py#L65

Reply:
Added